### PR TITLE
Register warden Egbdo

### PR DESCRIPTION
**Pull request by code423n4**
Auto-generated PR to register the new warden Egbdo

### CODE REVIEW

**Review by Egbodo — COMMENTED** [review]
Good work

**Review by Egbodo — COMMENTED** [review]
Good

**Review by Egbodo — APPROVED** [review]
Approve

**Comment by Egbodo** [comment]
Do

File: _data/handles/Egbdo.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Egbodo) [inline _data/handles/Egbdo.json:1]: Tilkg

Comment (Egbodo) [reply _data/handles/Egbdo.json:1]: Yiko

File: _data/handles/Egbdo.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "handle": "Egbdo",
+  "link": "",
```
Comment (Egbodo) [inline _data/handles/Egbdo.json:3]: You k m get t

**Review by Egbodo — CHANGES_REQUESTED** [review]
Go let

**Review by Egbodo — CHANGES_REQUESTED** [review]
Approve # # @axic

**Comment by SamusElderg** [comment]
Can not find a matching user in Discord, will leave the application open for now though

**Comment by Egbodo** [comment]
> Auto-generated PR to register the new warden Egbdo

**Review by Egbodo — APPROVED** [review]
Merge pull request

**Review by Egbodo — CHANGES_REQUESTED** [review]
Merge 

>     ###@`
Duplicate of #`

**Comment by SamusElderg** [comment]
> 

It doesn't work that way, for starters i would recommend checking into the `i-want-to-be-a-warden` channel in the Code4rena Discord channel and ensure you follow the instructions in the sticky messages as you might have missed them during the application process. Ensure your nickname in that discord channel is set as exactly `Egbdo` otherwise this PR will never be merged. Thanks

**Review by Egbodo — CHANGES_REQUESTED** [review]
Gm in

**Comment by SamusElderg** [comment]
Okay will process it now, looks all good, in the meantime please be patient and refrain from spamming the channels

**Comment by Egbodo** [comment]
> Can not find a matching user in Discord, will leave the application open for now though

It’s on discord

**Review by SamusElderg — APPROVED** [review]

**Review by Egbodo — APPROVED** [review]
Approve merge me

**Comment by SamusElderg** [comment]
> Approve merge me

As i said, please refrain from spamming, i have explained above that we are working on your application now 👍

**Pull request merged**